### PR TITLE
Buttons review

### PR DESCRIPTION
**Pull request by bruno-t-de-renzo**
- Adding the possibility to rotate and turn the screen on and off
- Refactoring of the "configuration reset" button

### CODE REVIEW

File: src/NerdMinerV2.ino.cpp [hunk 1]
```diff
@@ -1,8 +1,13 @@
+//Botón configuración
+#define PIN_BUTTON_1 0
+#define PIN_BUTTON_2 14
```
Comment (gaelbenoit) [inline src/NerdMinerV2.ino.cpp:3]: You should leave these lines in wManager.h

File: src/NerdMinerV2.ino.cpp [hunk 2]
```diff
@@ -31,6 +39,20 @@ unsigned long start = millis();
 
 //void runMonitor(void *name);
 
+void alternate_screen_state() {
+  int screen_state= digitalRead(38);
+  //Serial.printf("Screen state is '%s', switching to '%s'", screen_state, !screen_state);
+  Serial.println("Switching display state");
+  digitalWrite(38, !screen_state);
```
Comment (gaelbenoit) [inline src/NerdMinerV2.ino.cpp:46]: there is a macro defined for the back light pin : TFT_BL
```suggestion
  int screen_state= digitalRead(TFT_BL);
  //Serial.printf("Screen state is '%s', switching to '%s'", screen_state, !screen_state);
  Serial.println("Switching display state");
  digitalWrite(TFT_BL, !screen_state);
```

**Review by gaelbenoit — APPROVED** [review]
Hi, it's working great. It will help avoid killing the screen 😅

File: src/NerdMinerV2.ino.cpp (hunk 1 above)
Comment (bruno-t-de-renzo) [reply src/NerdMinerV2.ino.cpp]: Done

File: src/NerdMinerV2.ino.cpp (hunk 2 above)
Comment (bruno-t-de-renzo) [reply src/NerdMinerV2.ino.cpp]: Done

**Comment by BitMaker-hub** [comment]
Hi guys, yes I'll merge this functionalities. They are really interesting.
And on near future I'll use the first click to change new screens

**Review by BitMaker-hub — APPROVED** [review]
Evrything great, merged

**Pull request merged**